### PR TITLE
Update require RobloxGlobals.yaml

### PR DESCRIPTION
**Pull request by lack-of-gravity-jack**
## Changes

The description for "require" was very short and should provide more context on its behavior, especially regarding one of the major uses of modulescripts, that is to have a method of sharing values across scripts. Also does not elaborate on client server boundary affecting require

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/globals/RobloxGlobals.yaml [hunk 1]
```diff
@@ -180,28 +180,48 @@ functions:
...
+      and `Global.RobloxGlobals.require()` is called from both a `Class.LocalScript`
+      as well as a `Class.Script`, the code in the `Class.ModuleScript` will be run
+      twice, and the `Class.LocalScript` will receive a distinct return object from
+      the one received by the `Class.Script`.
 
-      If the ModuleScript the user wants to use has been uploaded to Roblox
-      (with the instance's name being 'MainModule'), it can be loaded by using
-      the require function on the asset ID of the ModuleScript, though only on
-      the server.
+      ALso note that if the `Class.ModuleScript` the user wants to run has been
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/globals/RobloxGlobals.yaml:205]:
```suggestion
      Also note that if the `Class.ModuleScript` the user wants to run has been
```

Comment (IgnisRBX) [inline content/en-us/reference/engine/globals/RobloxGlobals.yaml:205]:
```suggestion
      Also note that if the `Class.ModuleScript` the user wants to run has been
```

**Comment by IgnisRBX** [comment]
Minor edits to your contribution (mostly to make cross-links work). Please comment back here if everything looks good and accurate according to your proposal, and I'll merge the PR.

**Comment by lack-of-gravity-jack** [comment]
Looks good to me

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**